lib/server/server.js

```diff
@@ -568,18 +569,14 @@ function execute(port, options) {
     });
   });
 
-  if (options.watch) startLiveReload();
-  app.listen(port);
-
-  function startLiveReload() {
+  let numOfAttempts = 0;
```

Comment:
we only use this once. let's just remove this

Reply:
Sure 👍 , but IMO, it would confuse any contributor when we directly use `0` in function instead

Reply:
ok. let's leave it for now